AppendixFulcroErrorsAndWarnings.adoc

```diff
@@ -44,6 +44,8 @@ Could not find a DOM element with the given ID to mount to. Perhaps your HTML is
 [[err-comp-app-not-bound]]Cannot create proper fulcro component, as *app* isn't bound. This happens when something renders a Fulcro component outside of Fulcro's render context. See `with-parent-context`.::
 This can happen for example when you want a third-party component to render a Fulcro component. Wrap it with `with-parent-context` to preserve the important data.
 
+It also happens more rarely that this error appears in production code when you try to render a lazy-seq and then replace it with another one. See: https://github.com/fulcrologic/fulcro/discussions/477
```

Comment:
I would hint at the solution right here. E.g.:

> ...replace it with another one. To avoid the problem, you can force the lazy sequence e.g. by wrapping it with `vec`. See...

Reply:
I think that's too verbose, don't you think? 
Forcing a lazy-seq isn't a Fulcro thing. It's a Clojure thing.

Reply:
We need to be mindful of space in the logged messages. The whole purpose of this page is to be able to be more verbose, provide more insight, and suggest troubleshooting. So no, I do not think that is too verbose. Also, some Fulcro newcomers are obviously also quite new too Clojure so why not be a little nice to them?